NEWS.md

```diff
@@ -4,6 +4,18 @@ ClimaAtmos.jl Release Notes
 Main
 -------
 
+v0.27.2
+-------
+- Implements TOA edmf model
```

Comment:
```suggestion
- Extend GCM-driven single column to the top of atmosphere
```
(Sorry, that PR title is not accurate. Or you can just remove it from the news - it’s not that important for climaatmos users.)